src/components/tooltip/CallNode.js

```diff
@@ -99,8 +112,12 @@ export class TooltipCallNode extends React.PureComponent<Props> {
             }}
           />
         </div>
-        <div>{displayData.totalTimeWithUnit}</div>
-        <div>{displayData.selfTimeWithUnit}</div>
+        <div className="tooltipCallNodeImplementationTiming">
+          {displayData.totalTimeWithUnit}
+        </div>
+        <div className="tooltipCallNodeImplementationTiming">
+          {displayData.selfTimeWithUnit}
+        </div>
```

Comment:
Seeing `samples` everywhere is a bit too much: 
![image](https://user-images.githubusercontent.com/454175/84530809-cfad0d80-ace3-11ea-90dd-53ebb8450cf2.png)

What do you think of putting the unit in the header?

Reply:
I'm afraid it'll make too much whitespace:

<img width="479" alt="image" src="https://user-images.githubusercontent.com/1588648/85454586-b1a4a000-b562-11ea-986c-0cbc3845c37e.png">

I made a few tweaks to make it more readable, by aligning right, and adding a bit more padding.

<img width="820" alt="image" src="https://user-images.githubusercontent.com/1588648/85455499-90907f00-b563-11ea-840b-7d31419bf5b9.png">